user/APP/CAN_receive/CAN_receive.h

```diff
@@ -1,19 +1,21 @@
 /**
-  ****************************(C) COPYRIGHT 2016 DJI****************************
-  * @file       can_receive.c/h
-  * @brief      ���can�豸����շ�������ļ���ͨ��can�ж���ɽ���
-  * @note       ���ļ�����freeRTOS����
-  * @history
-  *  Version    Date            Author          Modification
-  *  V1.0.0     Dec-26-2018     RM              1. ���
-  *
-  @verbatim
-  ==============================================================================
-
-  ==============================================================================
-  @endverbatim
-  ****************************(C) COPYRIGHT 2016 DJI****************************
-  */
+  ******************************************************************************
+  * @file    APP/CAN_receive
+  * @date    15-January-2020
+  * @brief   This file contains functions to write to chassis and gimbal motors 
+	*          and read their information using CAN interrupt. 
+	* @attention The initialization of CAN channels must be done by calling funtions
+	*          in hardware/CAN.c. Additionally, the motor layout are as following:
+	*          CAN 1: chassis  M3508  
+	*									motor ID 1 2 3 4
+	*									write ID 0x200 (for all of them)
+	*                 read ID 0x201 0x202 0x203 0x204 
+	*					 CAN 2: gimbal M6020_yaw M6020_pitch P36_revolver P36_shoot
+	*                 motor ID 5 6 7 8
```

Comment:
Same thing with spacing

Reply:
This has been fixed in the latest commit